tests/fixtures/docker.py

```diff
@@ -72,6 +72,7 @@ def cleanup_all_new_docker_objects(docker: DockerClient, worker_id: str):
             logger.warning("Failed to clean up Docker objects")
 
 
+@pytest.mark.timeout(120)
```

Comment:
I've already addressed this one by moving it up to a `pytestmark`, let's just drop this line here:

```suggestion
```